packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -25,7 +25,18 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           [`.${IsFocusVisibleClassName} &:focus`]: {
             // Can't use getFocusStyle because it doesn't support wrapping links
             // https://github.com/OfficeDev/office-ui-fabric-react/issues/4883#issuecomment-406743543
-            outline: `1px solid ${theme.palette.neutralSecondary}`
+            // A box-shadow allows the focus rect to wrap links that span multiple lines
+            // and helps the focus rect avoid getting clipped.
+            boxShadow: `0 0 0 1px ${theme.palette.neutralSecondary} inset`
+          },
+          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
+            // For IE high contrast mode
+            borderBottom: 'none',
+            selectors: {
+              [`.${IsFocusVisibleClassName} &:focus`]: {
+                outline: `1px solid ${theme.palette.white}`
```

Comment:
@mikewheaton is this correct for HC?

Reply:
The color should be a keyword like `Window`, `WindowText`, or `Highlight`. Looking at our focus styles elsewhere, I think it should be `WindowText` in this case.

Reply:
The purpose of the keywords like `WindowText` is that they can be customized. Users have these configured to different colors based on their vision needs and general preferences. So I think it's best to respect that choice, even if it doesn't look the best. I'd rather have too much contrast than too little.